// Command
// python -m grpc_tools.protoc -I. --python_out=. --grpc_python_out=. GL20.proto

syntax = "proto3";

message Number {
    float value = 1;
}

message noMessage {}

service serviceGL20 {
    rpc SquareRoot(Number) returns (Number) {}
    rpc GL20_digitalWriteToggleAll(noMessage) returns (noMessage) {}
}